posthog/performance/sql.py

```diff
@@ -161,21 +160,12 @@ def clean_line(line: str) -> str:
 FROM {database}.kafka_performance_events
 """.format(
     columns=_column_names_from_column_definitions(columns),
-    target_table="writeable_performance_events",
+    target_table="sharded_performance_events",
```

Comment:
We should be writing to the distributed table still

Reply:
Ah, so https://github.com/PostHog/posthog/pull/13299#discussion_r1047779080 confuses me 😊 

Do we not create the table but still have it as the target?